### DISCUSSION TITLE
Nats.Net 2.0.0-rc.1

### DISCUSSION THREAD

**Discussion by mtmk** [nats-io/nats.net#222] [Announcements]
We're happy to announce our first release candidate 🎉 

### JetStream and Native AOT support

Nats.Net supports all new NATS JetStream, Object Store, Key/Value Store and Services APIs. We're also excited to support .NET 8.0 Native AOT deployments for your cloud-native workloads.

## What's Changed Since the last Beta
* Key/Value and Object Store compression in https://github.com/nats-io/nats.net.v2/pull/214
* JetStream API clean-up in https://github.com/nats-io/nats.net.v2/pull/212
* Auth error retry handling in https://github.com/nats-io/nats.net.v2/pull/217
* JetStream options API prefix bug fix in https://github.com/nats-io/nats.net.v2/pull/218
* List JetStream streams and consumers in https://github.com/nats-io/nats.net.v2/pull/219
* JetStream consume error notifications in https://github.com/nats-io/nats.net.v2/pull/220

**Full Changelog**: https://github.com/nats-io/nats.net.v2/compare/v2.0.0-beta.3...v2.0.0-rc.1

### NATS.Net

This is the main meta package containing all other packages except serialization.
```
dotnet add package NATS.Net --version 2.0.0-rc.1
```

If you are not deploying Native AOT you can also use ad-hoc JSON serialization:
```
dotnet add package NATS.Client.Serializers.Json --version 2.0.0-rc.1
```

### Individual Packages
```
dotnet add package NATS.Client.Core --version 2.0.0-rc.1
dotnet add package NATS.Client.Hosting --version 2.0.0-rc.1
dotnet add package NATS.Client.JetStream --version 2.0.0-rc.1
dotnet add package NATS.Client.KeyValueStore --version 2.0.0-rc.1
dotnet add package NATS.Client.ObjectStore --version 2.0.0-rc.1
dotnet add package NATS.Client.Services --version 2.0.0-rc.1
dotnet add package NATS.Client.Serializers.Json --version 2.0.0-rc.1
```

<hr /><em>This discussion was created from the release <a href='https://github.com/nats-io/nats.net.v2/releases/tag/v2.0.0-rc.1'>Nats.Net 2.0.0-rc.1</a>.</em>

**Comment by oising** [comment]
Congrats all!
